src/recursion.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function recursion(tree) {
+  const arr = [[]];
+  const level = 0;
+  rec(level, arr, tree);
```

Comment:
Better to give names that describe purpose of variable: for example not args but argumentsList